Makefile

```diff
@@ -228,9 +227,37 @@ add-license: addlicense
 
 # Build the OCI image
 .PHONY: oci-build
-oci-build: generate manifests manager test-envtest
-	BUILDAH_FORMAT=docker $(IMAGE_BUILDER) build -t $(OPERATOR_IMG) .
-
+oci-build: manifests generate fmt vet test-envtest
+	BUILDAH_FORMAT=docker $(IMAGE_BUILDER) build --build-arg TARGETOS=$(OS) --build-arg TARGETARCH=$(ARCH) -t $(OPERATOR_IMG) .
+
+# PLATFORMS defines the target platforms for the manager image to provide support to multiple
+# architectures. (i.e. make oci-buildx OPERATOR_IMG=quay.io/cryostat/cryostat-operator:2.3.0).
+# You need to be able to push the image for your registry (i.e. if you do not inform a valid value via OPERATOR_IMG=<myregistry/image:<tag>> than the export will fail)
+# If IMAGE_BUILDER is docker, you need to:
+# - able to use docker buildx. More info: https://docs.docker.com/build/buildx/
+# - have enable BuildKit, More info: https://docs.docker.com/develop/develop-images/build_enhancements/
+# If IMAGE_BUILDER is podman, you need to:
+# - install qemu-user-static.
+# To properly provided solutions that supports more than one platform you should use this option.
+PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
+.PHONY: oci-buildx
+oci-buildx: # manifests generate fmt vet test-envtest ## Build OCI image for the manager for cross-platform support
```

Comment:
Should these prerequisites be uncommented now?

Reply:
Ohh sorry totally forgot. I commented them out to skip steps.